### PR TITLE
update rules as of August 14

### PR DESCRIPTION
**Pull request by mlutze**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,63 @@
...
+The dealer draws the top card from the deck and lays it on the table to begin the pile.
+Play begins from the left of the dealer and continues clockwise.
+On their turn, a player must either:
+- play a card from their hand on the pile such that either the suit or the value of the card matches that of the top of the pile, or
+- draw a card from the deck.
+
+Either of these actions ends the player's turn.
+At any point, a player may accuse another of having broken a rule. Play pauses immediately.
+- If the player is found guilty of breaking a rule by consensus of players at the table, then the player must draw a card.
+- If the player is not found guilty of breaking a rule, then the accusing player must draw a card.
```
Comment (xdesou01) [inline README.md:15]: Should it be specified that play resumes once a penalty card is drawn (i.e., a player doesn't lose their turn if they are challenged and are either in favor of or against)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,63 @@
...
+You may play the trap card at any time.
+The trap card must change the suit (playing a Joker or playing on a Joker qualifies).
+If you play the trap card on your turn, your turn ends.
+
+## Prime Sequence
+_Added by Xavier, 14 Aug 2023_
+
+When you would play a card on your turn,
+you may instead play a collection of cards such that:
+- the first card alone would be valid to play
```
Comment (xdesou01) [inline README.md:49]: and prime

File: README.md [hunk 3]
```diff
@@ -1,3 +1,63 @@
...
+_Added by Matt, 14 Aug 2023_
+
+4 Jokers are added to the deck.
+They are considered wild:
+they may be played on any card and any card may be played on them.
+
+## Trap Card
+_Added by Andreas, 14 Aug 2023_
+
+On your turn, rather than playing or drawing a card, you may play a lay a trap card.
```
Comment (xdesou01) [inline README.md]: typo. Also maybe specify that the trap card must be face down. Did we also have a rule for not being allowed to look at the trap card after it gets layed?

Comment (xdesou01) [reply README.md]: (ignore the third part, just saw the other rule)

File: README.md [hunk 4]
```diff
@@ -1,3 +1,63 @@
...
+On their turn, a player must either:
+- play a card from their hand on the pile such that either the suit or the value of the card matches that of the top of the pile, or
+- draw a card from the deck.
+
+Either of these actions ends the player's turn.
+At any point, a player may accuse another of having broken a rule. Play pauses immediately.
+- If the player is found guilty of breaking a rule by consensus of players at the table, then the player must draw a card.
+- If the player is not found guilty of breaking a rule, then the accusing player must draw a card.
+
+If a player ends their turn with no cards, they win the round.
```
Comment (JonathanStarup) [inline README.md]: This might be weird later if we add out of turn rules

Comment (mlutze) [reply README.md]: I thought it would be interesting later. But I guess "if you ever run out of cards" is more fundamental?

Comment (JonathanStarup) [reply README.md]: Yea we already have it with trap cards actually

Comment (mlutze) [reply README.md]: Good point.

**Review by JonathanStarup — APPROVED** [review]

**Pull request merged**